### PR TITLE
Improved errors in Redis cache backend

### PR DESCRIPTION
**Pull request by Green-Cat**

### CODE REVIEW

File: phalcon/cache/backend/redis.zep [hunk 1]
```diff
@@ -111,14 +111,14 @@ class Redis extends Backend implements BackendInterface
 		}
 
 		if !success {
-			throw new Exception("Cannot connect to Redisd server");
+			throw new Exception("Could not connect to the Redisd server ".host.":".port);
```
Comment (sergeyklay) [inline phalcon/cache/backend/redis.zep]: Redisd → Redis

File: phalcon/cache/backend/redis.zep [hunk 2]
```diff
@@ -111,14 +111,14 @@ class Redis extends Backend implements BackendInterface
...
 		}
 
 		if fetch auth, options["auth"] {
 			let success = redis->auth(auth);
 
 			if !success {
-				throw new Exception("Redisd server is authentication failed");
+				throw new Exception("Failed to authenticate with the Redisd server");
```
Comment (sergeyklay) [inline phalcon/cache/backend/redis.zep]: Redisd → Redis

**Comment by sergeyklay** [comment]
It seems `d` it is `daemon`. Redis**d** == Redis Deamon

**Comment by Green-Cat** [comment]
Yeah redisd is the name of the redis sever process.

**Comment by andresgutierrez** [comment]
Thanks

**Pull request merged**